### ISSUE TITLE
Request to install a dll

### ISSUE THREAD

**Issue by Druidblack** [eebette/Jellyfin-Tools#8]
Good afternoon, install the utility? Each time the command is run, a dll installation request is made, a restart repeats the dll installation request again

```

root@omv:~# jellyfin-cover create --image /root/3.png --title Эфир
libraqm and/or fribidi dll's are not found! This package will not work without t                                                             hese dll's. Install them now? [y/N] (default: y)
dll archive downloaded successfully.
System is 64-bit: True
Will filter for dlls for architecture type: x64
Default Python directory detected at: /usr/bin
Extracting the downloaded dll's to the Python directory...
Installation Done! Please run your command again.
root@omv:~# ^C
root@omv:~# ^C
root@omv:~# jellyfin-cover create --image /root/3.png --title Эфир
libraqm and/or fribidi dll's are not found! This package will not work without these dll's. Install them now? [y/N] (default: y)
dll archive downloaded successfully.
System is 64-bit: True
Will filter for dlls for architecture type: x64
Default Python directory detected at: /usr/bin
Extracting the downloaded dll's to the Python directory...
Installation Done! Please run your command again.
root@omv:~#

```

**Comment by eebette** [comment]
Can you please run the following commands and provide their output? 

```sh
which python
```

It's odd that the Python directory is being detected as `/usr/bin`, instead of its own subdir. I think this might be an issue with your environment. 

For example, when I run this on my machine, I get: 

```sh
eric@eric-ubuntu:~$ which python
/usr/bin/python
```

**Comment by Druidblack** [comment]
```
root@omv:~# which python
/usr/bin/python
root@omv:~#

```

**Comment by eebette** [comment]
Thank you. Looks like there's a bug looking for the Python path. I'll implement a fix on my end.

**Comment by Druidblack** [comment]
Have you made any adjustments?

**Comment by eebette** [comment]
So, looking into this, the codebase is operating as expected and identifying where python is installed and moving the `libraqm.dll` and other `fribidi.dll` there as intended. Additionally, it is unexpected that Pillow packaged/shipped for Linux doesn't have these features readily available. 

I have 2 recommendations: 

### 1. Update the version of `jellyfin-tools` to the newly release `1.0.2` which has updated dependencies, and see if that possibly makes a difference. 

`pip install --upgrade jellyfin-tools`

### 2. It looks like this issue may occur when running on a Raspberry Pi or some other ARM based architecture. In that case, it is recommended that you install Pillow from source *after installing jellyfin-tools*

Discussion: https://github.com/python-pillow/Pillow/issues/6023#issuecomment-1030703549
Instructions: https://pillow.readthedocs.io/en/stable/installation/building-from-source.html

TL;DR

`sudo apt-get install python3-dev python3-setuptools`

```
sudo apt-get install libtiff5-dev libjpeg8-dev libopenjp2-7-dev zlib1g-dev \
    libfreetype6-dev liblcms2-dev libwebp-dev tcl8.6-dev tk8.6-dev python3-tk \
    libharfbuzz-dev libfribidi-dev libxcb1-dev
```

```
python3 -m pip install --upgrade pip
python3 -m pip install --upgrade Pillow --no-binary :all:
```

Good luck!

**Comment by Druidblack** [comment]
I am running your program on debian 11, which is installed on amd64 architecture (no arm processors). I updated your program and tried to install all the packages that you wrote. I still get a dll error.

May I ask you to package your program in a docker container and make a docker compose for installation?

**Comment by jspiers** [comment]
I'm getting the same error on an x86-based Macbook

**Comment by conradkun** [comment]
some on M1-Macbook

**Comment by eebette** [comment]
@Druidblack @jspiers @conradkun FYI Docker is supported now #25
